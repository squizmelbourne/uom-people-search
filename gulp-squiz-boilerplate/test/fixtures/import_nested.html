Nested Content
<!-- import:content test/fixtures/fragments/nest1.html -->