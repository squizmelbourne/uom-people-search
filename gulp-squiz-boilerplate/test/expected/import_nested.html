Nested Content
Nest1
Nest2